Cannot run daikoku with authenticated mongodb

Hi! :)

I try to run daikoku on a debian VM from docker.
Mongodb is installed on the linux machine (not via docker)
Authentication is activated in mongo
I created a database and a user for daikoku in mongodb.
The user has "readWrite" role on the daikoku db.

My run script
```
#!/bin/bash
docker run -p "8010:8080" maif/daikoku -Dconfig.file=/home/myuser/config/daikoku.conf -Dlogger.file=/home/myuser/config/daikoku.xml
```

When I run it, I get lots of the same warn log
```
[warn] r.c.n.ChannelFactory - [Supervisor-1/Connection-2] Skip channel init as host is null
[warn] r.c.n.ChannelFactory - [Supervisor-1/Connection-2] Skip channel init as host is null
[warn] r.c.n.ChannelFactory - [Supervisor-1/Connection-2] Skip channel init as host is null
[warn] r.c.n.ChannelFactory - [Supervisor-1/Connection-2] Skip channel init as host is null
[warn] r.c.n.ChannelFactory - [Supervisor-1/Connection-2] Skip channel init as host is null
```

Then, when I open the url http://myhost:8010
```
[error] daikoku-error-handler - Server Error [5849d51d-0a08-385e-9e08-6a5a1a351ec7]: MongoError['No primary node is available! (Supervisor-1/Connection-2)'] on /
reactivemongo.core.actors.Exceptions$PrimaryUnavailableException: MongoError['No primary node is available! (Supervisor-1/Connection-2)']
        at play.modules.reactivemongo.DefaultReactiveMongoApi.database(DefaultReactiveMongoApi.scala:41)
        at reactivemongo.api.MongoConnection.database(MongoConnection.scala:90)
Caused by: reactivemongo.core.actors.Exceptions$InternalState: null
        at reactivemongo.ChannelDisconnected(8578a166, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
        at reactivemongo.ChannelDisconnected(0e54aaff, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
        at reactivemongo.ChannelDisconnected(97cc1ff4, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
        at reactivemongo.ChannelDisconnected(76207738, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
        at reactivemongo.ChannelDisconnected(236959e5, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
        at reactivemongo.ChannelDisconnected(ca7677c8, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
        at reactivemongo.ChannelDisconnected(432d2e83, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
        at reactivemongo.ChannelDisconnected(84ce631d, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
        at reactivemongo.ChannelDisconnected(142e610e, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
        at reactivemongo.ChannelDisconnected(4760bacd, {{NodeSet None Node[localhost:27017: Unknown (0/10 available connections), latency=9223372036854775807, authenticated={}] }})(<time:1590863704614>)
```		
my daikoku conf : `/home/myuser/config/daikoku.conf` 

```
daikoku {

  mode = "prod"
  signingKey = "some key"

  exposedOn = ${http.port}

  api {
    type = "local"
    key = "some key"
    headerName = "Otoroshi-Claim"
    headerSecret = "some secret"
  }

  init {
    host = "localhost"
    admin {
      name = "Super admin"
      email = "someemail@gmail.com"
      password = "some password"
    }
    data {
      from = "/config/daikoku.ndjson"
      headers = {}
    }
  }

  snowflake {
    seed = 0
  }

  tenants {
    provider = "local"
  }

  exposition {
    provider = "none"
    otoroshi {
      stateHeaderName = "Otoroshi-State"
      stateRespHeaderName = "Otoroshi-State-Resp"
    }
  }

  otoroshi {
    groups {
      namePrefix = "daikoku"
    }
    sync {
      interval = 3600000
      master = false
      key = "secret"
      cron = true
      instance = 0
    }
  }

  stats {
    sync {
      interval = 3600000
      cron = true
    }
    call {
      interval = 600000
    }
  }
}

mongodb {
  uri = "mongodb://daikoku:somepassword@localhost:27017/daikoku"
}


http.port = 8080
https.port = disabled

play {
  application.loader = "fr.maif.otoroshi.daikoku.DaikokuLoader"
  modules {}
  filters {
    enabled = []
  }
  http {
    filters = play.api.http.NoHttpFilters
    secret {
      key = "some key"
    }
    session = {
      cookieName = "daikoku-session"
      secure = false
      maxAge = 259200000
      httpOnly = true
      sameSite = "lax"
      domain = null
      path = ${play.http.context}
      jwt {
        signatureAlgorithm = "HS256"
        expiresAfter = ${play.http.session.maxAge}
        clockSkew = 5 minutes
        dataClaim = "data"
      }
    }
  }
  server {
    akka {
      requestTimeout = 60s
    }
    http {
      port = ${http.port}
      idleTimeout = 60s
    }
  }
}
```


Database & user configuration in mongodb

```
use admin
db.auth('adminuser','someadminpassword')

use daikoku

// add a collection to create create db
db.dummy.insert({'dummy':'dummy')

db.createUser(
   {
     user: "daikoku",
     pwd: passwordPrompt(), 
     roles:
       [
         { role: "readWrite", db: "daikoku" }
       ]
   }
)
```

From bash I can connect succesfully to mongo database
```
mongo "mongodb://daikoku:somepassword@localhost:27017/daikoku"
MongoDB shell version v4.2.7
connecting to: mongodb://localhost:27017/daikoku?compressors=disabled&gssapiServiceName=mongodb
Implicit session: session { "id" : UUID("c80cff26-3184-4cf8-b2e5-6fb3e0320dcd") }
MongoDB server version: 4.2.7
```

--- a/daikoku/build.sbt
+++ b/daikoku/build.sbt
@@ -1,4 +1,6 @@
 import com.typesafe.sbt.packager.docker.{Cmd, ExecCmd}
+import com.typesafe.sbt.packager.docker.DockerChmodType
+import com.typesafe.sbt.packager.docker.DockerPermissionStrategy
 
 name := """daikoku"""
 organization := "fr.maif.otoroshi"
@@ -115,22 +117,16 @@ dockerExposedPorts := Seq(
   8080
 )
 packageName in Docker := "daikoku"
-
 maintainer in Docker := "MAIF OSS Team <oss@maif.fr>"
-
 dockerBaseImage := "openjdk:11-jre-slim"
-
 dockerUsername := Some("maif")
-
 dockerUpdateLatest := true
-
-dockerCommands :=
-  dockerCommands.value.flatMap {
-    case ExecCmd("ENTRYPOINT", args @ _*) =>
-      Seq(Cmd("ENTRYPOINT", args.mkString(" ")))
-    case v => Seq(v)
-  }
-
+dockerCommands := dockerCommands.value.filterNot {
+  case ExecCmd("CMD", args @ _*) => true
+  case cmd                       => false
+}
+dockerPackageMappings in Docker += (baseDirectory.value / "docker" / "start.sh") -> "/opt/docker/bin/start.sh"
+dockerEntrypoint := Seq("/opt/docker/bin/start.sh")
 dockerUpdateLatest := true
 
 // swaggerDomainNameSpaces := Seq("fr.maif.otoroshi.daikoku.domain")

--- a/daikoku/conf/base.conf
+++ b/daikoku/conf/base.conf
@@ -107,7 +107,7 @@ play {
   http {
     filters = play.api.http.NoHttpFilters
     secret {
-      key = "secret"
+      key = "snTgnSr806O%yII^mIYn@m!c8q#nVTdv9go%5swCyw4$!wxDqU41gI&iCb^HPMY2"
       key = ${?PLAY_CRYPTO_SECRET}
     }
     session = {

--- a/daikoku/docker/start.sh
+++ b/daikoku/docker/start.sh
@@ -1,0 +1,3 @@
+#!/bin/bash -e
+
+exec /opt/docker/bin/daikoku -Dplay.server.pidfile.path=/dev/null "$@"

